### ISSUE TITLE
Doubt in training

### ISSUE THREAD

**Issue by BRim28** [ilyak93/GAIN-pytorch#2]
Hi @ilyak93 

In the paper, I did not find any details indicating when to calculate the activation loss and the pixel level loss.
Should it happen in every epoch?
What is your go on this and why?

**Comment by ilyak93** [comment]
Hi, @BRim28 
By activation loss do you mean attention-mining loss ? 
Anyway, yeah just compute it as every other loss, like the classification loss.
You can also deduce from the formulas in the paper that the losses are summed,
including the classification loss, that should pretty much everything about when to use them.
Is that clear? 
If you look, in both mains (VOC and MedT) I calculate them each epoch.

**Comment by BRim28** [comment]
@ilyak93 

Thank you for the clarification.:+1:
